pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -232,7 +244,13 @@ stdenv.mkDerivation rec {
   ] ++ lib.optionals stdenv.isDarwin [
     # error: 'path' is unavailable: introduced in macOS 10.15
     "-DQT_FEATURE_cxx17_filesystem=OFF"
-  ] ++ lib.optional (qttranslations != null) "-DINSTALL_TRANSLATIONSDIR=${qttranslations}/translations";
+  ] ++ lib.optionals stdenv.hostPlatform.isMinGW [
+    "-DQT_HOST_PATH=${pkgsBuildBuild.qt6.qtbase}"
+    "-DQt6HostInfo_DIR=${pkgsBuildBuild.qt6.qtbase}/lib/cmake/Qt6HostInfo"
```

Comment:
If these paths are explicitly set, maybe there's no need to put it in depsBuildBuild? (Also causes inf recursion.)

Reply:
You are correct. Logic updated to only include this when stdenv.hostPlatform != stdenv.buildPlatform. Fixed.